Application/Tests/run_unix.bash

```
@@ -31,22 +31,119 @@ else
+            return 1
+        fi
+    fi
+    return 1
+  fi
+
+  #echo "All CSV files in $input_folder found matches in $output_folder (ignoring names)."
+  return 0
+}
+
+rm \"${WPWD}/corrected/test.settings\"
+CMD="${TGRABS} -d "${WPWD}" -i \"${WPWD}/test_frames/frame_%3d.jpg\" -o test -s \"${WPWD}/test.settings\" -auto_quit -nowindow -manual_matches {} -manual_splits {} -task convert -detect_type background_subtraction -match_mode automatic -history_matching_log history_matching_tgrabs.html"
```

Shellcheck (0.10.0)

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)